src/dynam.cpp

```diff
@@ -98,6 +102,50 @@ std::wstring Dynam::GetSymbolStr() const
     return Dynam::GetSymbolStr(m_symbolStr);
 }
 
+int Dynam::PrepareDynamEnclosure(FunctorParams *functoParams)
+{
+    if (this->HasEnclose()) {
+        Rend *open = new Rend();
+        Rend *close = new Rend();
+        std::wstring openElement, closeElement;
+        switch (this->GetEnclose()) {
+            case ENCLOSURE_brack: {
+                openElement.assign(L"[");
+                closeElement.assign(L"]");
+                break;
+            }
+            case ENCLOSURE_paren: {
+                openElement.assign(L"(");
+                closeElement.assign(L")");
+                break;
+            }
+            default: break;
+        }
+        // If both opening/closing element are set - add them to the start and end of the dynam. Apply normal fontstyle
+        // to both, so that brackets are not inclined
+        if (!openElement.empty() && !closeElement.empty()) {
+            // Add opening bracket to the start of the dynam
```

Comment:
Why don't you create the `*open` and `*close` Rend only here? No need to delete them otherwise